src/main/java/com/patchworkmc/manifest/converter/ModManifestConverter.java

```diff
@@ -121,8 +121,8 @@ private static JsonObject getDependencies(ModManifest manifest, ModManifestEntry
 			dependencyMap.get(mod.getModId()).forEach(c -> {
 				if (c.isMandatory() == mandatory) {
 					if (c.getModId().equals("forge")) {
-						// TODO depend on a specific version of API
-						deps.addProperty("patchwork", "*");
+						// TODO depend on a more specific version of API
+						deps.addProperty("patchwork", ">=0.4.0");
```

Comment:
0.4.* was before the snap
```suggestion
					deps.addProperty("patchwork", ">=0.5.0");
```